app/models/ems_event.rb

```diff
@@ -196,6 +196,7 @@ def get_target(target_type)
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
     target_type = "middleware_server"   if event.event_type == "hawkular_alert"
+    target_type = "container_node"      if event.event_type == "datawarehouse_event"
```

Comment:
these need to be renamed to datawarehouse_alert as @lucasponce pointed [here](https://github.com/ManageIQ/manageiq/pull/13669#discussion_r98056160)